### ISSUE TITLE
improve the accessibility of the website

### ISSUE THREAD

**Issue by robertoostenveld** [bids-standard/bids-website#68]
The feedback on [neurostars](https://neurostars.org/t/request-for-feedback-reorganizing-bids-website) included multiple people suggesting to move away from thewhite-on-black, and  @emdupre specifically mentioned accessibility at https://neurostars.org/t/request-for-feedback-reorganizing-bids-website/5476/3

For me personally [a11y](https://a11yproject.com/posts/a11y-and-other-numeronyms/) was a new term, but I figured it out by now.

**Comment by Remi-Gau** [comment]
I think this is still valid but needs to be made more specific to be actionable

**Comment by Remi-Gau** [comment]
OK closing for now
